const Error = () => {
  throw new Error("Something went wrong!")
}

export default Error
